charts/pega/charts/hazelcast/values.yaml

```diff
@@ -23,8 +23,10 @@ client:
   clusterName: "PRPC"
 # Server side settings for hazelcast
 server:
-  java_opts: "-Dhazelcast.initial.min.cluster.size=3 -Dhazelcast.jmx=true
-  -XX:MaxRAMPercentage=80.0 -XX:+HeapDumpOnOutOfMemoryError -XX:HeapDumpPath=/opt/hazelcast/logs/heapdump.hprof"
+  java_opts: "-XX:MaxRAMPercentage=80.0 -XX:+HeapDumpOnOutOfMemoryError -XX:HeapDumpPath=/opt/hazelcast/logs/heapdump.hprof
+  -XX:+UseParallelGC -Xlog:gc*,gc+phases=debug:file=/opt/hazelcast/logs/gc.log:time,pid,tags:filecount=5,filesize=3m"
+  min_cluster_size: "3"
```

Comment:
should this value to be "3" all the time? as I see we have ` .Values.server.min_cluster_size` exposed as part of same PR changes

Reply:
Yes. Earlier this value is passed using java_opts.